styleguide/source/_patterns/05-pages/news.twig

```diff
@@ -9,6 +9,8 @@
 {% endblock %}
 
 {% block pageContent %}
+  {% include '@atoms/paragraph.twig' %}
+  {% include '@atoms/paragraph.twig' %}
```

Comment:
It would be more efficient to do this as a `foreach` so that we can add as many paragraphs as is necessary and we don't have to update the `news.twig` if we need to update that data in the `.json`